test/jdk/java/foreign/StdLibTest.java

```diff
@@ -300,7 +315,7 @@ int rand() throws Throwable {
 
         int printf(String format, List<PrintfArg> args) throws Throwable {
             try (var arena = Arena.ofConfined()) {
-                MemorySegment formatStr = arena.allocateUtf8String(format);
+                MemorySegment formatStr = arena.allocateArray(ValueLayout.JAVA_BYTE, nativeBytesFromString(arena, format));
```

Comment:
This must change (I don't think it compiles as is):
```java
                MemorySegment formatStr = nativeBytesFromString(arena, format);
```
A similar change is needed on line 399.

Reply:
Sorry about this goof-up. When I tested the change to ensure functionality I had it implemented correctly, In pushing changes I used search and replace. I should have been more careful, apologize for this. 

Changed in https://github.com/ibmruntimes/openj9-openjdk-jdk21/compare/960e06a3679336e039a146ae1ebe4fcfa56da55c..a837b75d57e86bb113c40ee75d4738a6af6e1e8d